docs/release-notes/8.7.asciidoc

```diff
@@ -9,6 +9,11 @@
 [[known-issue-8.7.0]]
 ==== Known issues
 * After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
+* There is a known issue when upgrading Agents to 8.7.0 that are running Osquery.  It is possible that Agents can get stuck in an "Updating" state.  This is only known to occur if Agents have the Osquery Manager integration installed (https://github.com/elastic/elastic-agent/issues/2433[#2433]).  Users can do the following work around the issue.
```

Comment:
The affects 8.6.2 agents upgrading to 8.7.0, and will similarly affect 8.7.0 agents upgrading to future releases because it has the same logic.

Reply:
Thanks for pointing this out, @cmacknz. I can open a PR against the `8.6.2` release notes to make sure this gets added in.

Reply:
Confirmed this doesn't need to be doc'd in the 8.6.2 release notes because it only affects users moving from 8.6.2 to 8.7.0.